pkg/tempo/config.go

```diff
@@ -176,12 +176,23 @@ var DefaultRemoteWriteConfig = RemoteWriteConfig{
 	Compression: compressionGzip,
 }
 
+// TLSConfig controls the TLS config for the exporter's client.
+type TLSConfig struct {
+	CAFile             string `yaml:"ca_file,omitempty"`
+	CertFile           string `yaml:"cert_file,omitempty"`
+	KeyFile            string `yaml:"key_file,omitempty"`
+	InsecureSkipVerify bool   `yaml:"insecure_skip_verify,omitempty"`
+}
```

Comment:
What do you think about using Prometheus' struct directly? We can log an error if server_name was configured since OTel doesn't have an equivalent yet.

Reply:
Good idea, I missed it. Changed in c274aed102e775e0ab37c48d05b7d0c76f1e6197